mlir/lib/Target/LLVMIR/ModuleTranslation.cpp

```diff
@@ -894,6 +894,10 @@ LogicalResult ModuleTranslation::convertOneFunction(LLVMFuncOp func) {
     llvmFunc->addFnAttr(llvm::Attribute::getWithVScaleRangeArgs(
         getLLVMContext(), attr->getMinRange().getInt(),
         attr->getMaxRange().getInt()));
+  
+  // Add function attribute frame-pointer, if found.
+  if (mlir::LLVM::FramePointerKindAttr attr = func.getFramePointerAttr())
```

Comment:
```suggestion
  if (FramePointerKindAttr attr = func.getFramePointerAttr())
```
ultra nit: I think you can drop the namespace due to the using mlir::LLVM at the top of the file.